optimade/server/exception_handlers.py

```diff
@@ -82,7 +82,7 @@ def grammar_not_implemented_handler(request: Request, exc: VisitError):
     status = 501
     title = "NotImplementedError"
     detail = (
-        f"Error trying to process rule '{exc.tree.data}'"
+        f"Error trying to process rule '{exc}'"
```

Comment:
Did you see if we can get a better message here? I went [here](https://github.com/lark-parser/lark/blob/5346231e14d31ab5bbc3cbc014a31b405d40ef39/lark/exceptions.py#L99) to retrieve the passed "rule", but since it's updated, they don't store the rule in the exception anymore. What does this say when it raises now?
... Or did I already check the output with some tests?

Reply:
The output is checked, and as it stayed the same I decided not to probe any further!